### PR TITLE
Broadcast plugin and Channel Join 🤔

### PR DESCRIPTION
**Pull request by New-dev0**

### CODE REVIEW

File: bot/__init__.py [hunk 1]
```diff
@@ -58,6 +60,8 @@
 UN_BAN_COMMAND = get_config("UN_BAN_COMMAND", "unban")
 # start command
 START_COMMAND = get_config("START_COMMAND", "start")
+# broadcast command
+BROADCAST_COMMAND = get_config("BROADCAST_COMMAND","broadcast")
```
Comment (SpEcHiDe) [inline bot/__init__.py]: can you add space after comma, please? 😁

File: bot/plugins/broadcast.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# (c) Shrimadhav U K
```
Comment (SpEcHiDe) [inline bot/plugins/broadcast.py]: change this line.

Comment (New-dev0) [reply bot/plugins/broadcast.py]: Done?

**Comment by New-dev0** [comment]
Can u have a Check Now ?

**Comment by SpEcHiDe** [comment]
I didn't like the except Exception as e thing, anyways merging for now !!

**Pull request merged**